server/sonar-db-dao/src/main/java/org/sonar/db/component/BranchDto.java

```diff
@@ -126,30 +116,26 @@ public BranchDto setKeeType(BranchKeyType t) {
   }
 
   /**
-   * This is the getter used by MyBatis mapper. It does
-   * not handle the special value used to map null field.
+   * This is the getter used by MyBatis mapper.
    */
   private String getKee() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getKee" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)